plugin/trino-google-sheets/src/main/java/io/trino/plugin/google/sheets/SheetsErrorCode.java

```diff
@@ -27,7 +27,9 @@
     SHEETS_BAD_CREDENTIALS_ERROR(0, EXTERNAL),
     SHEETS_METASTORE_ERROR(1, EXTERNAL),
     SHEETS_UNKNOWN_TABLE_ERROR(2, USER_ERROR),
-    SHEETS_TABLE_LOAD_ERROR(3, INTERNAL_ERROR);
+    SHEETS_TABLE_LOAD_ERROR(3, INTERNAL_ERROR),
+    SHEETS_INSERT_ERROR(4, INTERNAL_ERROR),
+    SHEETS_COLUMN_TYPE_NOT_SUPPORTED(5, USER_ERROR);
```

Comment:
Replace the usage of `SHEETS_COLUMN_TYPE_NOT_SUPPORTED` with StandardErrorCode's `NOT_SUPPORTED`.

Also, please change like below so that we can get clean diff when adding a new error code in the future change
```java
    SHEETS_INSERT_ERROR(4, INTERNAL_ERROR),
    /**/;
```